baseframe/static/js/baseframe-bootstrap.js

```diff
@@ -292,6 +292,28 @@ window.Baseframe.Forms = {
         }
       }
     });
+  },
+  handleAjaxPost: function(config) {
```

Comment:
Can you please add a comment that explains what this function does and what attributes are excepted/accepted in `config`?

Reply:
Added